### DISCUSSION TITLE
Add a settings or shortcut or button to choice whether looking for answers based on your codebase.

### DISCUSSION THREAD

**Discussion by wxwok** [gitbito/bitoai#214] [Ideas]
Every time I ask Bito through the Clion plugin, it asks me: Are you looking for answers based on your codebase? 
I rarely ask questions about my code, and sometimes, my code is too big to send to Bito, which makes Bito unable to answer the questions.  There are some delays between my questions and Bito's questions.
So it is only efficient sometimes to enable this feature. Can a feature be added to the Bito plugin to allow users to use the "look for answers based on your code" optional or by shortcut or by clicking a button.

**Comment by nisha-bito** [comment]
Hi @wxwok ,

Thanks for your feedback. We will take this feedback.

thanks
Nisha
